shell/platform/android/test/io/flutter/embedding/android/FlutterFragmentActivityTest.java

```diff
@@ -180,8 +180,8 @@ public void itReturnsValueFromMetaDataWhenCallsShouldHandleDeepLinkingCase3()
     Bundle bundle = new Bundle();
     FlutterFragmentActivity spyFlutterActivity = spy(activity);
     when(spyFlutterActivity.getMetaData()).thenReturn(bundle);
-    // Empty bundle should return false.
-    assertFalse(spyFlutterActivity.shouldHandleDeeplinking());
+    // Empty bundle should return true.
+    assertTrue(spyFlutterActivity.shouldHandleDeeplinking());
```

Comment:
can you have test to verify the flag set to false will cause shouldHandleDeeplinking to return false?

Reply:
The test already exist in L161

Reply:
sorry I am not sure how I missed this